sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/ClassicTableTypeMapping.java

```diff
@@ -69,7 +72,8 @@ public ClassicTableTypeMapping() {
   public String[] mapToHiveType(String clientTypeName) {
     Collection<String> hiveTableType = clientToHiveMap.get(clientTypeName.toUpperCase());
     if (hiveTableType == null) {
-      LOG.warn("Not supported client table type " + clientTypeName);
+      LOG.warn("Not supported client table type {}",
+        MDC.of(LogKeys.CLIENT_TABLE_TYPE$.MODULE$, clientTypeName));
```

Comment:
NIt: `CLIENT_TABLE_TYPE` is a bit confusing here. Is there a better name for this?

Reply:
`CLIENT_TABLE_TYPE ` -> `CLASSIS_TABLE_TYPE`